vignettes/generic.Rmd

```diff
@@ -72,6 +72,7 @@ generic_derivations <- tibble::tribble(
   "derive_vars_merged()",           "selection",   "variables", "single",
   "derive_vars_extreme_event()",    "selection",   "variables", "multiple",
   "derive_vars_computed()",         "computation", "variables", "single",
+  "derive_vars_cat()",              "selection",   "variables", "single",
```

Comment:
"selection" is correct, right?

Reply:
Not really because new values are created instead of selecting an existing one.

I would classify the new functions rather as "Special Variable-Adding Functions" than as "Generic Variable-Adding Functions" (see cheatsheet). Thus I wouldn't add it here.